src/lib/service-error-translator/service-error-translator.tsx

```diff
@@ -29,6 +31,17 @@ interface ServiceErrorProps {
   errorCode: number;
 }
 
+export const serviceErrorI18nInstance = i18nInstance
+
+export const serviceErrorAddI18nResource = (
```

Comment:
will it be better if we place the verb in the beginning such as `addI18nServiceErrorResource`?

Reply:
I see, my initial consideration is that I want to emphasize that those two variables belongs to `Service Error`. But I think your suggestion easy to understand as well, so do I need to change those two variable to:

```
i18nInstanceServiceError
addI18nServiceErrorResource
```
what do you think?

Reply:
or maybe `addI18nResourceServiceError` will be better?

Reply:
i think we can leave `serviceErrorI18nInstance` as it is since it's not a function. but for the other one, it depends on where `resource` belongs to, is it `serviceError` or `i18n`? it's kinda confusing

Reply:
all right then, the `resource` will be belongs to `i18n`  since the function is meant to inject `i18n` language resource from external source to `service error` utils